src/main/java/seedu/address/logic/commands/util/CommandUtil.java

```diff
@@ -62,6 +65,7 @@ public static ProfileItem getProfileItem(Model model, Index profileItemIndex) th
      * @return Feedback message of the operation result for display.
      */
     public static CommandResult getCommandResult(Model model, String message, TabName tabName) {
+        assert model != null && message != null && tabName != null;
```

Comment:
I think requireAllNonNull would be better here

Reply:
Thanks for the advice, did not know there is such a thing.